src/vs/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -141,26 +142,62 @@ export class WelcomePageAction extends Action {
 interface ExtensionSuggestion {
 	name: string;
 	title?: string;
+	description?: string;
 	id: string;
 	isKeymap?: boolean;
 	isCommand?: boolean;
+	isExtensionPack?: boolean;
+	icon?: string;
+	link?: string;
+	extensionPackExtensions?: any[];
 }
 
+
 const extensionPacks: ExtensionSuggestion[] = [
-	{ name: localize('welcomePage.javaScript', "JavaScript"), id: 'dbaeumer.vscode-eslint' },
-	{ name: localize('welcomePage.python', "Python"), id: 'ms-python.python' },
-	// { name: localize('welcomePage.go', "Go"), id: 'lukehoban.go' },
-	{ name: localize('welcomePage.php', "PHP"), id: 'felixfbecker.php-pack' },
-	{ name: localize('welcomePage.azure', "Azure"), title: localize('welcomePage.showAzureExtensions', "Show Azure extensions"), id: 'workbench.extensions.action.showAzureExtensions', isCommand: true },
-	{ name: localize('welcomePage.docker', "Docker"), id: 'ms-azuretools.vscode-docker' },
+	{
+		name: localize('welcomePage.adminPack', "SQL Admin Pack"),
+		title: localize('welcomePage.showAdminPack', "SQL Admin Pack"),
+		description: 'Admin Pack for SQL Server is a collection of popular database administration extensions to help you manage SQL Server',
+		id: 'microsoft.admin-pack',
+		extensionPackExtensions: [
+			{
+				name: 'SQL Server Agent',
+				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-agent-extension?view=sql-server-2017',
+				icon: '../../../workbench/contrib/welcome/defaultExtensionIcon.svg'
+			},
+			{
+				name: 'SQL Server Profiler',
+				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-profiler-extension?view=sql-server-2017',
+				icon: '../../../workbench/contrib/welcome/defaultExtensionIcon.svg'
+			},
+			{
+				name: 'SQL Server Import',
+				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-import-extension?view=sql-server-2017',
+				icon: '../../../workbench/contrib/welcome/defaultExtensionIcon.svg'
+			},
+			{
+				name: 'SQL Server Dacpac',
+				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-dacpac-extension?view=sql-server-2017',
+				icon: '../../../workbench/contrib/welcome/defaultExtensionIcon.svg'
+			}
+		],
+		isExtensionPack: true
+	},
 ];
 
-const keymapExtensions: ExtensionSuggestion[] = [
-	{ name: localize('welcomePage.vim', "Vim"), id: 'vscodevim.vim', isKeymap: true },
-	{ name: localize('welcomePage.sublime', "Sublime"), id: 'ms-vscode.sublime-keybindings', isKeymap: true },
-	{ name: localize('welcomePage.atom', "Atom"), id: 'ms-vscode.atom-keybindings', isKeymap: true },
+const extensions: ExtensionSuggestion[] = [
+	{ name: localize('welcomePage.powershell', "Powershell"), id: 'microsoft.powershell', description: 'Develop PowerShell scripts in Azure Data Studio', icon: 'https://raw.githubusercontent.com/PowerShell/vscode-powershell/master/images/PowerShell_icon.png', link: 'https://docs.microsoft.com/en-us/sql/azure-data-studio/powershell-extension?view=sql-server-ver15' },
```

Comment:
For the links in the page make sure to remove the en-us parts.

Reply:
removed